src/ekobox/info_file.py

```diff
@@ -79,5 +79,5 @@ def build(
         dtype=float,
     )
     template_info["AlphaS_Vals"] = alphas_values.tolist()
-    template_info["AlphaS_Qs"] = np.array(operators_card.mu2grid, dtype=float).tolist()
+    template_info["AlphaS_Qs"] = operators_card.mu2grid
```

Comment:
```suggestion
    template_info["AlphaS_Qs"] = operators_card.mu2grid.tolist()
```
In the current and upcoming version of EKO this is an array, so at least the `.tolist()` was actually required.

Reply:
It seems that in the test (at least) it is a grid because it was failing before. I can show you in the next commit

Reply:
Ah, weird...

Reply:
So this is the reason of [03c9bcd](https://github.com/NNPDF/eko/pull/229/commits/03c9bcd2de603e8d08ba53c185a8fec7031059d3)